lib/App/Config/Chronicle.pm

```diff
@@ -110,6 +110,29 @@ has setting_name => (
     default  => 'settings1',
 );
 
+=head2 refresh_interval
+
+How much time (in seconds) should pass between C<check_for_update> invocations until
+it actually will do (a bit heavy) lookup for settings in redis.
+
+Default value is 10 seconds
+
+=cut
+
+has refresh_interval => (
```

Comment:
I think this would need to be added under https://github.com/basiliscos/perl-App-Config-Chronicle/blob/0e1333370716122f76c816b52cec6ee43fb4cf88/lib/App/Config/Chronicle.pm#L272 and that's a general issue with the API here: we seem to be creating class attributes which we can use for Moose validation.

This means that any changes like this could technically break existing apps: we can't ever add new methods or attributes to this class. Since we don't expect there to be many users at the moment, we might be able to get away with it, but I'd suggest an API update that either includes a `get_` prefix for keys, or splits the class into an empty namespace that can deal with user attribute validation. That class would still have several reserved keys, including:

* new
* has
* BUILD
* DESTROY